draft-ietf-quic-recovery.md

```diff
@@ -990,17 +989,20 @@ congestion without depending on PTO expiration.
 
 ### Establishing Persistent Congestion
 
-A sender establishes persistent congestion on receiving an acknowledgement if at
-least two ack-eliciting packets are declared lost, and:
+A sender establishes persistent congestion after the receipt of an
```

Comment:
I think it is better to write _after_. The check for persistent congestion might happen directly when receiving an ack, but it might also happen after the loss detection timer fires.